src/util/gas-factory-helpers.ts

```diff
@@ -56,15 +56,25 @@ export async function getHighestLiquidityV3NativePool(
 ): Promise<Pool | null> {
   const nativeCurrency = WRAPPED_NATIVE_CURRENCY[token.chainId as ChainId]!;
 
-  const nativePools = _([FeeAmount.HIGH, FeeAmount.MEDIUM, FeeAmount.LOW])
+  const nativePools = _([
+    FeeAmount.HIGH,
+    FeeAmount.MEDIUM,
+    FeeAmount.LOW,
+    FeeAmount.LOWEST,
```

Comment:
Some chains may not have `FeeAmount.LOWEST`, however it should be fine as they will be filtered out